### DISCUSSION TITLE
UX/UI: Something around notification

### DISCUSSION THREAD

**Discussion by ccoVeille** [forem/forem#20937] [Feature Requests]
Hi

I would like to share something that I thought about and that could be valuable for forem/dev.to

I love the way a comment from the owner/poster is displayed

![Screenshot_20240509_161007_Forem.jpg](https://github.com/forem/forem/assets/3875889/cfd019be-1013-4c79-9682-12bb39959cba)

I would love seeing this information in the notifications.

![Screenshot_20240509_161229_Forem.jpg](https://github.com/forem/forem/assets/3875889/673503a3-0722-4227-9be0-e3ae1dd7d541)

Right now, I have to open the post to check if the like comes from post owner.

Also, I would love if we could get information in notifications when the post owner like one of our replies, it's not the same as the like made by another user.

![Screenshot_20240509_150537_Forem.jpg](https://github.com/forem/forem/assets/3875889/1ae40949-36ec-4ea9-b797-04c3a763909c)

**Comment by ccoVeille** [comment]
I summon @michael-tharrington because I'm sure he likes me, I have a proof.  😂🤣

![Screenshot_20240509_161629_Forem.jpg](https://github.com/forem/forem/assets/3875889/f9ac8328-e2ff-4ce4-8173-c332c9431f1f)

Yes, I know I abuse our online relationship based on fun and beard.

https://dev.to/michaeltharrington/comment/2f143

**Reply by michael-tharrington** [reply]
Haha! I like this one. It's a cool idea!

**Reply by ccoVeille** [reply]
Of course, it's a good idea! 😅🤣

An easy one, I think ( _famous last words_ ™️)
